ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -266,7 +370,9 @@ func validateRequest(w http.ResponseWriter, request types.NetworkFaultRequest, r
 	return nil
 }
 
-func getTaskMetadataForFault(w http.ResponseWriter, agentState state.AgentState, requestType string, r *http.Request) (*state.TaskResponse, error) {
+// validateTaskMetadata will check the associated task metadata to make sure the task has enabled fault injection
```

Comment:
nit(non-blocking): perhaps we should also include that we're obtaining the task metadata in addition to validating it

Reply:
Updated.